src/frontend/qt_sdl/EmuSettingsDialog.cpp

```diff
@@ -196,6 +208,9 @@ void EmuSettingsDialog::done(int r)
 
             Config::ConsoleType = consoleType;
             Config::DirectBoot = directBoot;
+            
+            Config::Slot2Addon = chosenaddon;
+    				NDS::SetSlot2Addon(Config::Slot2Addon);
```

Comment:
Tabs here to convert.

Reply:
Almost forgot about that! Thanks for reminding me!